src/traits.jl

```diff
@@ -56,9 +56,11 @@ hue(c::Union{C,AlphaColor{C},ColorAlpha{C}}) where {C<:Union{Lab,DIN99,DIN99o,DI
 hue(c::Union{C,AlphaColor{C},ColorAlpha{C}}) where {C<:Luv} =
     (h = atand(c.v, c.u); h < 0 ? h + 360 : h)
 
+# fallbacks for compN
+_comp(::Val{N}, c::Colorant) where N = getfield(c, N)
+_comp(::Val{N}, c::AlphaColor) where N = getfield(c, N + 1)
+_comp(::Val{N}, c::AlphaColorN{N}) where N = alpha(c)
```

Comment:
We may need a parametric interface like this, not for fallback. In fact, The "ugly" tuple `comp_n = (comp1, comp2, comp3, comp4, comp5)` was used in `_show_components()`.

However, when users overload the getters, I think the nonparametric methods (e.g. `comp1`) are simpler. (`comp1` might already have been overloaded somewhere. :ghost:) So I think `comp1` etc. should be left as independent methods, not as aliases.